internal/manifest/manifest.go

```diff
@@ -128,6 +60,23 @@ func (p Policy) Hash() HexString {
 
 // Validate checks the validity of all fields in the reference values.
 func (r ReferenceValues) Validate() error {
+	if r.AKS != nil {
+		if err := r.AKS.Validate(); err != nil {
+			return fmt.Errorf("validating AKS reference values: %w", err)
+		}
+	}
+
+	if r.BareMetalTDX != nil {
+		if err := r.BareMetalTDX.Validate(); err != nil {
+			return fmt.Errorf("validating bare metal TDX reference values: %w", err)
+		}
+	}
+
+	return fmt.Errorf("no reference values provided")
```

Comment:
This fails unconditionally regardless of whether one or more set of reference values was set.